### PR TITLE
Delete "uppercase" keyboard name, create new "lowercase" keyboard

### PR DESCRIPTION
**Pull request by sh-xguitar**
## Types of Changes

- [x] Keymap/layout/userspace (addition or update)
- [x] Keyboard (addition or update)
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
I would also add that the dashes should be changed to underscores in the file/path names.

File: keyboards/ASH-1800/keymaps/ANSI-WKL/keymap.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        writePinHigh(CAPS_LOCK_LED_PIN);
+    } else {
+        writePinLow(CAPS_LOCK_LED_PIN);
+    }
+    if (IS_LED_ON(usb_led, USB_LED_SCROLL_LOCK)) {
+        writePinHigh(SCROLL_LOCK_LED_PIN);
+    } else {
+        writePinLow(SCROLL_LOCK_LED_PIN);
+    }
+}
```
Comment (drashna) [inline keyboards/ASH-1800/keymaps/ANSI-WKL/keymap.c:56]:
```suggestion
bool led_update_user(led_t led_state) {
    // writePin sets the pin high for 1 and low for 0.
    // In this example the pins are inverted, setting
    // it low/0 turns it on, and high/1 turns the LED off.
    // This behavior depends on whether the LED is between the pin
    // and VCC or the pin and GND.
    writePin(NUM_LOCK_LED_PIN, !led_state.num_lock);
    writePin(CAPS_LOCK_LED_PIN, !led_state.caps_lock);
    writePin(SCROLL_LOCK_LED_PIN, !led_state.scroll_lock);
    
    return false;
}
```

File: keyboards/ASH-1800/keymaps/ISO-WK/keymap.c [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        writePinHigh(CAPS_LOCK_LED_PIN);
+    } else {
+        writePinLow(CAPS_LOCK_LED_PIN);
+    }
+    if (IS_LED_ON(usb_led, USB_LED_SCROLL_LOCK)) {
+        writePinHigh(SCROLL_LOCK_LED_PIN);
+    } else {
+        writePinLow(SCROLL_LOCK_LED_PIN);
+    }
+}
```
Comment (drashna) [inline keyboards/ASH-1800/keymaps/ISO-WK/keymap.c:56]:
```suggestion
bool led_update_user(led_t led_state) {
    // writePin sets the pin high for 1 and low for 0.
    // In this example the pins are inverted, setting
    // it low/0 turns it on, and high/1 turns the LED off.
    // This behavior depends on whether the LED is between the pin
    // and VCC or the pin and GND.
    writePin(NUM_LOCK_LED_PIN, !led_state.num_lock);
    writePin(CAPS_LOCK_LED_PIN, !led_state.caps_lock);
    writePin(SCROLL_LOCK_LED_PIN, !led_state.scroll_lock);
    
    return false;
}
```

File: keyboards/ASH-1800/keymaps/ISO-WKL/keymap.c [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        writePinHigh(CAPS_LOCK_LED_PIN);
+    } else {
+        writePinLow(CAPS_LOCK_LED_PIN);
+    }
+    if (IS_LED_ON(usb_led, USB_LED_SCROLL_LOCK)) {
+        writePinHigh(SCROLL_LOCK_LED_PIN);
+    } else {
+        writePinLow(SCROLL_LOCK_LED_PIN);
+    }
+}
```
Comment (drashna) [inline keyboards/ASH-1800/keymaps/ISO-WKL/keymap.c:56]:
```suggestion
bool led_update_user(led_t led_state) {
    // writePin sets the pin high for 1 and low for 0.
    // In this example the pins are inverted, setting
    // it low/0 turns it on, and high/1 turns the LED off.
    // This behavior depends on whether the LED is between the pin
    // and VCC or the pin and GND.
    writePin(NUM_LOCK_LED_PIN, !led_state.num_lock);
    writePin(CAPS_LOCK_LED_PIN, !led_state.caps_lock);
    writePin(SCROLL_LOCK_LED_PIN, !led_state.scroll_lock);
    
    return false;
}
```

File: keyboards/ASH-XIIX/ash-xiix.c [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+  // put your per-action keyboard code here
+  // runs for every action, just before processing by the firmware
+  return process_record_user(keycode, record);
+}
+void led_set_kb(uint8_t usb_led) {
+  // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+  led_set_user(usb_led);
+}
+*/
```
Comment (drashna) [inline keyboards/ASH-XIIX/ash-xiix.c:42]:
```suggestion
*/

bool led_update_kb(led_t led_state) {
    bool res = led_update_user(led_state);
    if(res) {
        // writePin sets the pin high for 1 and low for 0.
        // In this example the pins are inverted, setting
        // it low/0 turns it on, and high/1 turns the LED off.
        // This behavior depends on whether the LED is between the pin
        // and VCC or the pin and GND.
        writePin(NUM_LOCK_LED_PIN, !led_state.num_lock);
        writePin(CAPS_LOCK_LED_PIN, !led_state.caps_lock);
        writePin(SCROLL_LOCK_LED_PIN, !led_state.scroll_lock);
    }
    
    return res;
}
```

File: keyboards/ASH-1800/keymaps/ANSI-WK/keymap.c [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        writePinHigh(CAPS_LOCK_LED_PIN);
+    } else {
+        writePinLow(CAPS_LOCK_LED_PIN);
+    }
+    if (IS_LED_ON(usb_led, USB_LED_SCROLL_LOCK)) {
+        writePinHigh(SCROLL_LOCK_LED_PIN);
+    } else {
+        writePinLow(SCROLL_LOCK_LED_PIN);
+    }
+}
```
Comment (drashna) [inline keyboards/ASH-1800/keymaps/ANSI-WK/keymap.c:56]:
```suggestion
bool led_update_user(led_t led_state) {
    // writePin sets the pin high for 1 and low for 0.
    // In this example the pins are inverted, setting
    // it low/0 turns it on, and high/1 turns the LED off.
    // This behavior depends on whether the LED is between the pin
    // and VCC or the pin and GND.
    writePin(NUM_LOCK_LED_PIN, !led_state.num_lock);
    writePin(CAPS_LOCK_LED_PIN, !led_state.caps_lock);
    writePin(SCROLL_LOCK_LED_PIN, !led_state.scroll_lock);
    
    return false;
}
```

**Pull request closed without merging**